verilog/analysis/default_rules.h

```diff
@@ -33,6 +33,7 @@ constexpr const char* kDefaultRuleSet[] = {
     "generate-label",
     "generate-label-prefix",
     "always-comb",
+    "disable-non-sequential",
```

Comment:
I think we should start with it default off and have people try it before we switch it on by default.